tests.py

```diff
@@ -0,0 +1,194 @@
+import os
+import json
+import responses
+import unittest
+from review_app_status import ReviewAppStatus
+from unittest import mock
+
+import review_app_status
+
+
+class TestReviewAppStatus(unittest.TestCase):
+    def setUp(self):
+        self.rpa = ReviewAppStatus(
+            "1",
+            {
+                "pull_request": {"head": {"sha": "commitsha12345"}},
+                "repository": {"deployments_url": "https://foo.bar/deployments"},
+            },
+            [200, 302],
+        )
+
+    @responses.activate
+    @mock.patch.dict(os.environ, {"GITHUB_TOKEN": "secret"})
+    def test_get_deployments(self):
+        response_body = [{"name": "deployment1"}]
+        responses.add(
+            responses.GET,
+            "https://foo.bar/deployments",
+            body=json.dumps(response_body),
+            status=200,
+        )
+
+        with self.assertLogs(level="INFO") as log:
+            self.rpa._get_deployments()
+
+        self.assertEqual(len(responses.calls), 1)
+        self.assertEqual(responses.calls[0].request.url, "https://foo.bar/deployments")
+        self.assertIn(
+            '{"name": "deployment1"}', str(responses.calls[0].response.content)
+        )
+        self.assertEqual(len(log.output), 1)
```

Comment:
The "Java-style" asserts are considered a bad practice these days. Instead (when you have the luxury of using pytest) you can do this:

assert len(log.output) == 1

Reply:
Changed. Can we discuss the reason during the developer session?